src/Forms/GridField/GridFieldVersionedState.php

```diff
@@ -11,7 +11,11 @@ class GridFieldVersionedState implements GridField_ColumnProvider
 {
     protected $column = null;
 
-    protected $versionedLabelFields = ['Name', 'Title'];
+    protected $versionedLabelFields = [];
+
+    public function __construct($versionedLabelFields = []) {
```

Comment:
lets have `[ 'Title' ]` as the default, as all `DataObject` have that field defined.

Reply:
👍